src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const changeTable = Array.from(table.rows);
+
+changeTable.forEach((element) => {
+  const secondColumnClone = element.cells[1].cloneNode(true);
+
+  const lastCell = element.cells[element.cells.length - 1];
```

Comment:
Ensure that 'element.cells' is not empty to avoid errors when accessing the last cell.